### DISCUSSION TITLE
Love the Vanilla-like feel but a few small suggesgeons.

### DISCUSSION THREAD

**Discussion by Tibbitts3** [BeansGalaxy/Beans-Backpacks-2#111] [Feedback]
My one issue is how the backpack interacts with Elytra. I have mods that allow for Elytra customization and I have a cape, so its disappointing that the Elytra backpack combination not only dis-allows backpack trims, but also limits the Elytra to being blank. I think a good solution would be to make the "Winged Backpack" unique that it does not come with wings, but is instead compatible with them in the Chestplate slot, or allow any backpack to be worn in combination with Elytra via the config (maybe this is possible already I don't fully understand configs so I wouldn't know). Either way I really love the mod and I'm going to be using it regardless.

**Comment by BeansGalaxy** [comment]
Yeah... the elytra customization is something I'm not 100% happy with yet, Originally in my head elytra are a similar material to leather so I made they dyable but from a standpoint of progression, by the time you have elytra you also have the materials for trims. Also the way the dyed elytra wings are rendered cannot be compatible with mods like Elytra Trims.

As for player capes, I can easily check what cape a player has and use a different texture. I haven't implemented it tho because I'm not sure what I want to do next for the elytra. I think it would be interesting to see dyed winged backpacks with trims as well, I would need to rethink how to render the elytra to work with those mods and how I'm rendering it now is the best way with my current understanding.

I hope to sort all this out in time since an elytra is a big accomplishment in every world so thanks for being patient.
